browser/src/libs/phabricator/backend.tsx

```diff
@@ -359,19 +360,18 @@ async function convertConduitRepoToRepoDetails(repo: ConduitRepo): Promise<Phabr
 }
 
 function convertToDetails(repo: ConduitRepo): PhabricatorRepoDetails | null {
-    let uri: ConduitURI | undefined
-    for (const u of repo.attachments.uris.uris) {
-        const normalPath = u.fields.uri.normalized.replace('\\', '')
-        if (normalPath.startsWith(window.location.host + '/')) {
-            continue
-        }
-        uri = u
-        break
-    }
-    if (!uri) {
+    const enabledURIs = repo.attachments.uris.uris
+        // Filter out disabled URIs
+        .filter(({fields}) => !fields.uri.disabled)
+        .map(({ fields }) => ({
+            isExternalURI: !fields.uri.normalized.replace('\\', '').startsWith(window.location.host + '/'),
+            rawURI: fields.uri.raw
+        }))
+    if (!enabledURIs.length) {
```

Comment:
just a legibility thing, feel free to ignore if you don't like it

```suggestion
    if (enabledURIs.length === 0) {
```